### DISCUSSION TITLE
[FR] Possibility to set a country/region to a site settings for multi-region sites

### DISCUSSION THREAD

**Discussion by tibbis** [craftcms/cms#16171] [Ideas] [answered]
Possibility to set a country/region to a site in craft site settings as an addition to language: 

<img width="394" alt="388131829-d0cc2938-b085-438c-a2f2-5b23078f33ff" src="https://github.com/user-attachments/assets/92d30ecf-e328-4ed2-b9a6-f2780c6fb319">

You can now only set language, but not possible to combine any lang with a region for multi-region+multi-lang sites.

**Comment by brandonkelly** [comment, marked as answer]
Having sites represent different countries is only one of several use cases for multi-site, so I’m not sure this makes sense to add. Probably would cause confusion in most cases.

**Reply by tibbis** [reply]
Then as counter suggestion is to be able to extend the countries in the language dropdown. Then this would not be needed. For example `en-SE` exists but not `en-FR` which is a bigger population. I tried using `extraAppLocales` but didnt help. 

<img width="367" alt="Screenshot 2024-11-23 at 16 52 25" src="https://github.com/user-attachments/assets/cb3684c3-a23e-4b5b-8197-4f696a989287">

**Reply by brandonkelly** [reply]
You can add custom locales using the [localeAliases](https://craftcms.com/docs/5.x/reference/config/general.html#localealiases) config setting.

```php
->localeAliases([
    'en-FR' => [
        'aliasOf' => 'en',
        'displayName' => 'English (French)',
    ],
]);
```
